cli.py

```
@@ -228,6 +277,8 @@ def main() -> None:
         run_simulation()
     elif args.command == "classify":
         run_classifier()
```

Ruff (0.14.10)

Undefined name `run_classifier`

(F821)